app/code/Magento/Ui/Model/Export/SearchResultIterator.php

```diff
@@ -40,23 +42,23 @@ public function key()
     /**
      * @return void
      */
-    public function next()
+    public function next(): void
     {
         next($this->items);
     }
 
     /**
      * @return void
      */
-    public function rewind()
+    public function rewind(): void
     {
         reset($this->items);
     }
 
     /**
      * @return bool
      */
-    public function valid()
+    public function valid(): bool
```

Comment:
```suggestion
    #[\ReturnTypeWillChange]
    public function valid()
```

Reply:
Hello @ihor-sviziev
This is really true.
This is big BIC IMHO.
We fixed this changes.
@karyna-tsymbal-atwix thank you for the fast fix.